src/clj/metafacture_playground/process.clj

```diff
@@ -36,6 +36,7 @@
 
 (defn- flux->flux-content [flux fix morph output]
   (-> flux
+      (clj-str/replace #"(^|\s)//.*\n" "")
```

Comment:
This doesn't cover something like `| encode-json//(prettyPrinting="false")` (no blank before the `//`), right?

Approving since that's consistent with the syntax coloring in the VSC extension (does not color the comment for that example), however that kind of comment does actually (seem to) work in a flux, so we should probably open an issue for that. If the implementation here and in the extension are completely independent, it might make sense to already change it here.

Reply:
I move the corresponding issue back to 'working' and test which way is better.